mangaki/mangaki/tests/test_work.py

```diff
@@ -61,6 +62,32 @@ def setUp(self):
             'nsfw_ok': True
         })
 
+    def test_search(self):
+        accents = self.create_anime(title='Un titre accentué oui oui')
+        no_accents = self.create_anime(title='Un titre sans accents non non')
+        tests = [
+            ('medaka', self.manga),
+            ('titre', [accents, no_accents]),
+            ('titre oui', accents),
+            ('titre non', no_accents),
+            ('accentue', accents),
+            ('accent', [accents, no_accents]),
+            ('boxers', None),
+        ]
+        for query, expected in tests:
+            qs = list(Work.objects.filter(title__search=query))
+            if expected is None:
+                self.assertFalse(qs)
+                continue
+            elif isinstance(expected, (list, tuple)):
+                self.assertEqual(len(qs), len(expected))
+                for w in qs:
```

Comment:
I prefer not to introduce possible weird cases with zip & different length (even though right now we ensure that `qs` and `expected` have the same length). There is value in keeping tests as simple as possible.